.nycrc

```diff
@@ -0,0 +1,17 @@
+{
+  "lines": 91,
+  "functions": 56,
```

Comment:
When I ran this locally on node 7, it failed with 55.something - could we drop this to `55` just to avoid the failure later, when I add node 7 to travis?